plugins/scaffolder/src/components/TemplateCard/TemplateCard.tsx

```diff
@@ -94,13 +108,27 @@ export const TemplateCard = ({ template }: TemplateCardProps) => {
           classes={{ root: classes.title }}
         />
       </CardMedia>
-      <CardContent>
+      <CardContent style={{ display: 'grid' }}>
+        <Box className={classes.box}>
+          <Typography variant="body2" className={classes.label}>
+            Description
+          </Typography>
+          {templateProps.description}
+        </Box>
+        <Box className={classes.box}>
+          <Typography variant="body2" className={classes.label}>
+            Owner
+          </Typography>
+          <Typography variant="inherit">{templateProps.owner}</Typography>
```

Comment:
You shouldn't use the `owner` field here directly, but instead query the relations.

There is a helper you can use:
```typescript
const ownedByRelations = getEntityRelations(entity, RELATION_OWNED_BY)
```

Note that this can return multiple owners so you have to take care displaying all of them. `EntityRefLinks` helps you here:
```typescript
<EntityRefLinks
  entityRefs={ownedByRelations}
  defaultKind="group"
/>
```

Reply:
@Fox32 I tried with that method but always returns me empty, like there were no relations bonded to templates

Reply:
This is due to https://github.com/backstage/backstage/issues/5440 .

Would you like to add those lines to the `BuiltinKindsEntityProcessor`, so that we do emit those relations? That would solve two problems at once, since you'll be able to use the relations like @Fox32 suggests.

Reply:
Yeah sure @freben, I'll add them